general/press-and-publications.rst

```diff
@@ -0,0 +1,132 @@
+Press and Publications
+======================
+
+In this page, we aim to provide you links to presentations and popular
+blog posts about OpenWISP.
+
+Presentations:
+--------------
+
+`PyCon Italia 2017 <https://www.pycon.it/en/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=tm7Opg3QyZk
+- slides: `<https://www.pycon.it/conference/talks/applying-the-unix-
+  philosophy-to-django-projects-a-report-from-the-real-world>`_
+
+`OpenWRT Summit 2017 <http://openwrtsummit.org>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=n531yTtJimU
+- slides: http://static.nemesisdesign.net/openwisp2-openwrt-summit-2017/
+
+`FOSDEM 2017 <https://fosdem.org/2019/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- abstract: https://archive.fosdem.org/2017/schedule/event/openwisp2
+
+`IETF Meeting 103 Bangkok <https://www.ietf.org>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- `<https://datatracker.ietf.org/meeting/103/materials/slides-103-gaia-
+  openwisp-a-hackable-network-management-system-for-the-21st-centry-00>`_
+
+`django-freeradius at PyCon Italy 2018 <https://www.pycon.it/it/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*talk by Fiorella De Luca*
+
+- video: https://www.youtube.com/watch?v=Yapdso_6EGA
+- abstract: https://www.pycon.it/conference/talks/django-freeradius
+
+`Opening Proprietary Networks with OpenWISP <https://2017.djangocon.eu/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*lightning talk at DjangoCon Europe 2017*
+
+- slides: `<https://www.slideshare.net/FedericoCapoano/opening-propietary-
+  networks-with-openwisp>`_
+
+`Do you really need to fork OpenWRT? <http://openwrtsummit.org>`_
```

Comment:
do not add the link in the heading, link the phrase `OpenWRT Summit` below. Applies to all the other lines, if the title of the event is not written after the heading, add it please.